### PR TITLE
Add unit tests for Toponym query handlers

### LINKED ISSUE #58: Write unit tests for MeadiatR/Toponyms

### PR DESCRIPTION
**Pull request by ErikSavchynAI**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Summary of issue

Unit tests were needed for the Toponym query handlers to ensure their proper functionality and to validate various scenarios including success, failure, and edge cases.

## Summary of change

Added unit tests for the GetById, GetByStreetcodeId, and GetAll MediatR Toponym query handlers. The tests cover success and failure scenarios, edge case handling, and verification of mapping and repository interactions.

## Testing approach

Unit tests were written to validate functionality and coverage for all relevant Toponym query handlers, ensuring thorough testing of multiple scenarios.

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: Streetcode/Streetcode.XUnitTest/BLL/MediatRTests/Toponyms/GetAll/GetAllToponymsHandlerTests.cs [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        // Act
+        var result = await _handler.Handle(query, CancellationToken.None);
+
+        // Assert
+        result.IsSuccess.Should().BeTrue();
+        result.Value.Toponyms.Should().BeEmpty();
+        VerifyMocksCalledOnce();
+    }
+
+    // ---------- helpers ----------
```
Comment (DenysSaiuk) [inline Streetcode/Streetcode.XUnitTest/BLL/MediatRTests/Toponyms/GetAll/GetAllToponymsHandlerTests.cs]: You can comment it "// Arrange" too. But it's up to you.

**Review by DenysSaiuk — APPROVED** [review]
Perfect code. Everything is ok.

**Review by Ilbko — APPROVED** [review]
